<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>471</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>837</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionCreate"/>
   </widget>
   <widget class="QMenu" name="Font_Style">
    <property name="title">
     <string>Style</string>
    </property>
    <addaction name="Change_Font_Style"/>
    <addaction name="Change_Font_Size"/>
    <addaction name="Change_Font_Color"/>
    <addaction name="Change_Text_Backqround_Color"/>
    <addaction name="separator"/>
    <addaction name="Change_Background_Color"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="Font_Style"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionCreate">
   <property name="text">
    <string>Create</string>
   </property>
  </action>
  <action name="Change_Font_Style">
   <property name="text">
    <string>Change Font Style</string>
   </property>
  </action>
  <action name="Change_Font_Size">
   <property name="text">
    <string>Change Font Size</string>
   </property>
  </action>
  <action name="Change_Font_Color">
   <property name="text">
    <string>Change Font Color</string>
   </property>
  </action>
  <action name="Change_Background_Color">
   <property name="text">
    <string>Change Background Color</string>
   </property>
  </action>
  <action name="Change_Text_Backqround_Color">
   <property name="text">
    <string>Change Backqround Text Color</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
